packages/runtime-core/src/componentPublicInstance.ts

```diff
@@ -192,7 +191,7 @@ export type ComponentPublicInstance<
   $emit: EmitFn<E>
   $el: any
   $options: Options & MergedComponentOptionsOverride
-  $forceUpdate: ReactiveEffect
+  $forceUpdate: () => void
```

Comment:
This should be the public type and it doesn't take any arguments / return anything